src/main/java/Duke.java

```diff
@@ -34,19 +31,51 @@ public static void main(String[] args) {
             } else if (line.equals("list")) {
                 output = "    ____________________________________________________________\n";
                 for (int i = 0; trackIndex > i; i++) {
-                    String record = "     " + (i + 1) + ".["
-                            + toDo[i].getStatusIcon() + "] "+ toDo[i].getDescription() + "\n";
+                    String record = "     " + (i + 1) + "." + toDo[i].toString() + "\n";
                     output = output.concat(record);
                 }
                 output = output.concat("    ____________________________________________________________\n");
             } else if (line.contains("done")) {
                 int dividerPosition = line.indexOf(" ");
+                //What if the number is more than 10?
                 int numberInTaskArray = Integer.parseInt(line.substring(dividerPosition + 1)) - 1;
                 toDo[numberInTaskArray].setIsDone();
                 output = "____________________________________________________________\n"
                         + "     Nice! I've marked this task as done: \n"
                         + "       [X] " + toDo[numberInTaskArray].getDescription() + "\n"
                         + "    ____________________________________________________________";
+            } else if (line.contains("todo")) {
+                //Way too many lines, why not store how the output will look like as a method in the class
+                String description = line.substring(5);
+                toDo[trackIndex] = new Todo(description);
+                output = "    ____________________________________________________________\n"
+                        + "     Got it. I've added this task:\n"
+                        + "      " + toDo[trackIndex].toString() + "\n"
+                        + "     Now you have " + (trackIndex + 1)  + " tasks in the list.\n"
+                        + "    ____________________________________________________________";
+                trackIndex++;
+            } else if (line.contains("deadline")) {
+                int dividerPosition = line.indexOf("/");
+                String description = line.substring(9, dividerPosition - 1);
+                String date = line.substring(dividerPosition + 4);
+                toDo[trackIndex] = new Deadline(description, date);
+                output = "    ____________________________________________________________\n"
+                        + "     Got it. I've added this task:\n"
+                        + "      " + toDo[trackIndex].toString() + "\n"
+                        + "     Now you have " + (trackIndex + 1) + " tasks in the list.\n"
+                        + "    ____________________________________________________________";
```

Comment:
Maybe can write a separate function for writing output? So as to not make each output assignment so long.